fetch_rapids.cmake

```diff
@@ -11,6 +11,8 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 # =============================================================================
+set(rapids-cmake-repo bdice/rapids-cmake)
+set(rapids-cmake-branch cuco-cccl-2.2.0)
```

Comment:
I will revert this before merging. Depends on https://github.com/NVIDIA/cuCollections/pull/404/ merging, and rapids-cmake being updated to use that commit in `versions.json`.